pkg/logger/output_builder.go

```diff
@@ -73,40 +73,43 @@ func (o osFileSystem) OpenFile(name string, flag int, perm os.FileMode) (*os.Fil
 }
 
 type OutputBuilder interface {
-	Build(output LogOutputType, logFilePath, defaultLogFilePath string) io.Writer
+	Build(output LogOutputType, logFilePath string) (io.Writer, error)
 }
 
 type outputBuilder struct {
 	logCategory LogCategory
 	fileSystem  FileSystem
+	defaultFile string
```

Comment:
Should this be named `defaultFile` or simply `logFile`?

Reply:
I think `defaultFile`, since the purpose of this one is to have the default destination file in hand in case a provided custom file errors, knowing that the default logging directories and files should have been created at product installation.